conftest.py

```diff
@@ -486,47 +486,12 @@ def fixture_ported_to_in_jvm(request, fixture_dtest_setup):
             if skip_msg:
                 pytest.skip(skip_msg)
 
+
 def _skip_ported_msg(current_running_version, ported_from_version):
     if loose_version_compare(current_running_version, ported_from_version) >= 0:
         return "ported to in-JVM from %s >= %s" % (ported_from_version, current_running_version)
 
 
-@pytest.fixture(autouse=True)
-def fixture_ported_to_in_jvm(request, fixture_dtest_setup):
```

Comment:
@adelapena @ekaterinadimitrova2 This was a character-for-character duplicate of the one above :)